.github/workflows/test.yml

```diff
@@ -11,13 +11,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 19
```

Comment:
nit: feel free to ignore but any reason to prefer 19 over 18? I'd generally prefer the stable even numbered versions.